recipes/pdal/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+class PdalConan(ConanFile):
+    name = "pdal"
+    description = "PDAL is Point Data Abstraction Library. GDAL for point cloud data."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://pdal.io"
+    license = "BSD-3-Clause"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
```

Comment:
If pdal does not offer shared builds at all as you said in https://github.com/conan-io/hooks/pull/260 then why have you added `False` as an option here? Maybe you forgot the remove it?

Reply:
Is a boolean option allowed to have only one value? I think it can break something. There is a possibility that static build will work someday so I didn't remove `shared` option at all.  `tbb` recipe has a binary flag.

Reply:
I think one possible value should work fine. Other not used values aren't part of the package ID, hence not breaking any existing packages if we add later on a `False` option